tests/messages/test_traction_qc_message.py

```diff
@@ -22,34 +22,38 @@ class TestTractionQcMessage:
     @pytest.fixture()
     def valid_traction_qc_message(self):
         traction_qc_message = TractionQcMessage()
-        traction_qc_message.requests(0).supplier_sample_name = "supplier_sample_name_DDD"
-        traction_qc_message.requests(0).container_barcode = "FD20706500"
-        traction_qc_message.requests(0).sheared_femto_fragment_size = "5"
-        traction_qc_message.requests(0).post_spri_concentration = "10"
-        traction_qc_message.requests(0).post_spri_volume = "20"
-        traction_qc_message.requests(0).final_nano_drop_280 = "280"
-        traction_qc_message.requests(0).final_nano_drop_230 = "230"
-        traction_qc_message.requests(0).final_nano_drop = "200"
-        traction_qc_message.requests(0).shearing_qc_comments = "Comments"
-        traction_qc_message.requests(0).date_submitted_utc = datetime.now().timestamp() * 1000
-
-        traction_qc_message.requests(1).supplier_sample_name = "supplier_sample_name_DDD2"
-        traction_qc_message.requests(1).container_barcode = "FD20706501"
-        traction_qc_message.requests(1).sheared_femto_fragment_size = "9"
-        traction_qc_message.requests(1).post_spri_concentration = "10"
-        traction_qc_message.requests(1).post_spri_volume = "30"
-        traction_qc_message.requests(1).final_nano_drop_280 = "180"
-        traction_qc_message.requests(1).final_nano_drop_230 = "130"
-        traction_qc_message.requests(1).final_nano_drop = "100"
-        traction_qc_message.requests(1).shearing_qc_comments = ""
-        traction_qc_message.requests(1).date_submitted_utc = datetime.now().timestamp() * 1000
+
+        request = traction_qc_message.create_request()
+        request.sanger_sample_id = "sanger_sample_id_DDD"
+        request.container_barcode = "FD20706500"
+        request.sheared_femto_fragment_size = "5"
+        request.post_spri_concentration = "10"
+        request.post_spri_volume = "20"
+        request.final_nano_drop_280 = "280"
+        request.final_nano_drop_230 = "230"
+        request.final_nano_drop = "200"
```

Comment:
Could you confirm that representing numbers as strings is correct use?

Reply:
Yes they are strings.  You can see that here: https://github.com/sanger/tol-lab-share/pull/64/files#diff-f45fc30e279785a3bd04969b44b9dcfeb0f8242150600a9257a6c7c2c1269a93R24-R33